results/prepare_data.py

```diff
@@ -802,3 +822,64 @@ def previous_doc_refferal_result(value, fwb):
         fwb.append(Spacer(1, 2 * mm))
 
     return fwb
+
+
+def table_part_result(value):
+    try:
+        value = json.loads(value)
+    except:
+        return None
+
+    if not value:
+        return None
+
+    styleSheet = getSampleStyleSheet()
+    style = styleSheet["Normal"]
+    style.fontName = "FreeSans"
+    style.fontSize = 8
+    style.alignment = TA_JUSTIFY
+
+    table_titles = value['columns']['titles']
+    table_settings = value['columns']['settings']
+
+    opinion = [[Paragraph(f"{t}", style) for t in table_titles]]
+
+    table_rows = value['rows']
+    for t in table_rows:
+        temp_data = [[Paragraph(f"{row_data}", style) for row_data in t]]
+        opinion.extend(temp_data)
```

Comment:
Думаю стоит append просто сделать
```suggestion
        opinion.extend([Paragraph(f"{row_data}", style) for row_data in t])
```